SimPEG/EM/Utils/AnalyticUtils.py

```diff
@@ -1,4 +1,5 @@
-from SimPEG import Mesh, np, Utils
+import numpy as np
+from SimPEG import Mesh, Utils
```

Comment:
That actually is probably a better idea. For now it likely isn't a big problem though. If you want that done, I can get on that.